docs/connectors/building-connectors/developing-destination-connectors.mdx

```diff
@@ -21,24 +19,39 @@ This file provides the main functionality of your Destination Connector.
     }
     ```
 
-- **`NewDestination()`**: A constructor function for your Destination struct. Note that this is the same function that should be set as the value of `Connector.NewDestination`. The constructor should be used to wrap your Destination in the default middleware. You can add additional middleware, but unless you have a very good reason, you should always include the default middleware.
+## Destination Connector Lifecycle Functions
 
-    ```go
-    func NewDestination() sdk.Destination {
-        // Create Destination and wrap it in the default middleware.
-        return sdk.DestinationWithMiddleware(&Destination{}, sdk.DefaultDestinationMiddleware()...)
-    }
-    ```
+### `NewDestination()`
+
+A constructor function for your Destination struct. Note that this is the same function that should be set as the value of `Connector.NewDestination`. The constructor should be used to wrap your Destination in the default middleware. You can add additional middleware, but unless you have a very good reason, you should always include the default middleware.
+
+```go
+func NewDestination() sdk.Destination {
+    // Create Destination and wrap it in the default middleware.
+    return sdk.DestinationWithMiddleware(
+      &Destination{}, 
+      sdk.DefaultDestinationMiddleware(opts ...DestinationMiddlewareOption)...,
```

Comment:
I find this a bit confusing, it's not working Go code. I'd remove the options here and just rely on the part below that mentions the middleware options, maybe mention that those options can be supplied to `sdk.DefaultDestinationMiddleware` to configure the default parameters.